src/components/ClusterNodes/NodeDetailsList.tsx

```diff
@@ -369,6 +369,7 @@ export default function NodeDetailsList({
                 onClick={() => {
                     column.isSortingAllowed && handleSortClick(column)
                 }}
+                data-testid={`${column.label}`}
```

Comment:
We can directly write it as
```suggestion
                data-testid={column.label}
```